frontend/src/config/haiti/layers.json

```diff
@@ -0,0 +1,3668 @@
+        "color": "#cb181d"
+      },
+      {
+        "value": 56,
+        "label": "56C - 58C",
+        "color": "#a50026"
+      },
+      {
+        "value": 58,
+        "label": "58C - 60C",
+        "color": "#ad3a00"
+      }
+    ]
+  },
+  "lst_anomaly": {
+    "title": "Land Surface Temperature - 10-day Anomaly (MODIS)",
+    "type": "wms",
+    "server_layer_name": "myd11a2_txd_dekad",
+    "base_url": "https://api.earthobservation.vam.wfp.org/ows/",
+    "additional_query_params": {
+      "styles": "tdd_21_m12_12"
+    },
+    "date_interval": "days",
+    "opacity": 0.7,
+    "legend_text": "Daytime land surface temperature anomaly",
+    "legend": [
+      {
+        "value": -100,
+        "label": "< -12C",
+        "color": "#05284b"
+      },
+      {
+        "value": -12,
+        "label": "-12C to -10C",
+        "color": "#1d2e83"
+      },
+      {
+        "value": -10,
+        "label": "-10C to -8C",
+        "color": "#24479d"
+      },
+      {
+        "value": -8,
+        "label": "-8C to -7C",
+        "color": "#2166ac"
+      },
+      {
+        "value": -7,
+        "label": "-7C to -6C",
+        "color": "#1d8dbe"
+      },
+      {
+        "value": -6,
+        "label": "-6C to -5C",
+        "color": "#34a9c3"
+      },
+      {
+        "value": -5,
+        "label": "-5C to -4C",
+        "color": "#6cb4de"
+      },
+      {
+        "value": -4,
+        "label": "-4C to -3C",
+        "color": "#7ecae7"
+      },
+      {
+        "value": -3,
+        "label": "-3C to -2C",
+        "color": "#90e0ef"
+      },
+      {
+        "value": -2,
+        "label": "-2C to -1C",
+        "color": "#c3f6ff"
+      },
+      {
+        "value": -1,
+        "label": "-1C to +1C",
+        "color": "#fafafa"
+      },
+      {
+        "value": 1,
+        "label": "+1C to +2C",
+        "color": "#fddbc7"
+      },
+      {
+        "value": 2,
+        "label": "+2C to +3C",
+        "color": "#fcbba1"
+      },
+      {
+        "value": 3,
+        "label": "+3C to +4C",
+        "color": "#fc9272"
+      },
+      {
+        "value": 4,
+        "label": "+4C to +5C",
+        "color": "#fb6a4a"
+      },
+      {
+        "value": 5,
+        "label": "+5C to +6C",
+        "color": "#ef3b2c"
+      },
+      {
+        "value": 6,
+        "label": "+6C to +7C",
+        "color": "#d73027"
+      },
+      {
+        "value": 7,
+        "label": "+7C to +8C",
+        "color": "#cb181d"
+      },
+      {
+        "value": 8,
+        "label": "+8C to +10C",
+        "color": "#b2182b"
+      },
+      {
+        "value": 10,
+        "label": "+10C to +12C",
+        "color": "#a50026"
+      },
+      {
+        "value": 12,
+        "label": "> +12C",
+        "color": "#67000d"
+      }
+    ]
+  },
+  "vac_food_insec": {
+    "title": "Food insecurity prevalence (ZimVAC 2020)",
+    "type": "admin_level_data",
+    "path": "data/zimbabwe/zim_vac_food_insec_2020.json",
```

Comment:
@wadhwamatic there are a few layer definitions referencing Zimbabwe files in here. It's probably worth removing them for now to clean up. Or would you like to keep them as examples?

Reply:
was a copy/paste job. I'll remove them